src/main/java/core/basesyntax/strategy/BalanceHandler.java

```diff
@@ -1,11 +1,17 @@
 package core.basesyntax.strategy;
 
+import core.basesyntax.service.StorageService;
 import core.basesyntax.service.implementation.StorageServiceImpl;
 
 public class BalanceHandler implements FruitShopOperationsHandler {
+    private StorageService storageService;
+
+    public BalanceHandler() {
+        storageService = new StorageServiceImpl();
```

Comment:
You shouldn't create service by hands in constructor but accept it as constructor parameter. By creating your service here via `new` operator you are depending on concrete implementation of `StorageService`, it's violation of DI principle. Fix it in all handlers.